src/encoded/audit/antibody_lot.py

```diff
@@ -32,72 +32,51 @@ def audit_antibody_missing_characterizations(value, system):
         return
```

Comment:
This would be a little tighter if it checked if ANY of the elements of the array are 'control'

`if ‘control’ in [ t.get(‘investigated_as’) for t in value[‘targets’] ]:`

*should* work (untested)